config/samples/emqx/v2alpha2/rebalance.yaml

```diff
@@ -1,8 +1,9 @@
-apiVersion: apps.emqx.io/v1beta4
+apiVersion: apps.emqx.io/v2alpha2
```

Comment:
Copy this file, don't move this file, we need keep this file to be v1beta4 example